content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -50,6 +50,9 @@ folder i.e. `gdrive://<base>/path/to/folder`. The base can be one of:
    and [shared drives](https://support.google.com/a/users/answer/9310351)\*
    (these two can only be referenced by ID).
 
+   > ⚠️ Folders should be shared to specific users (or groups) in order for them
```

Comment:
Folders -> Folder

Reply:
Hmmm I think it's better in plural:

> ...including shared folders...
> Folders should be shared...

Reply:
_Folder ID_ - means you specify a single folder. And sharing should be setup for that folder.

Reply:
I would just change this if it didn't make a difference but actually the form in which the context is written I think implies plural here:

> There's a few alternatives to construct a GDrive remote URL... such as... shared **folders** [plural]... The base can be one of:
> 1. _Folder ID_... identifier for every Google Drive folder, including shared **folders** [plural]
> > **Folders** should [plural, referring to the above mentions]

But I guess if it was "The folder" it would also make sense (referring to the folder which ID we are using — a bit indirect but OK). No strong opinion really, up to you 🙂

Reply:
To be precise we should communicate the following:

Folder that corresponds to the _Folder ID_ mentioned must be shared to specific users (account emails) or groups in order for them to ...

Reply:
O_o I see it has a typo! "in order for them to access them with DVC" should've been just "in order to access them..." maybe that's what was confusing? Fixed now...

Reply:
> To be precise we should communicate:
Folder that corresponds to the Folder ID...

OK, I think that was implied. But I've re-written it now so it's extra explicit. PTAL